src/styles/utilities/spacing.js

```diff
@@ -195,6 +199,26 @@ export default {
         paddingHorizontal: 20,
     },
 
+    ph6: {
+        paddingHorizontal: 24,
+    },
+
+    ph7: {
```

Comment:
Is this used? If not, let's remove it and any other unused classes that have been added here. For example, we have ph2 and ph4 above but not ph3 since it isn't used.

Reply:
Ideally we should keep all the values even if they aren't used.
This won't increase the bundle size as unwanted objects will be tree shaken during compile time

Reply:
Bundle size isn't the reason we want to avoid adding unused classes, the main reason is to avoid adding unnecessary code that can potentially be used by other contributors. In this case, we don't have a need for ph7, but if it is added in this PR then somebody could use ph7 later on and cause inconsistent styles across the app.

Reply:
Done

Reply:
Were these removed? I still see them in the project now, it looks like 7-10 are unused after the latest changes.